### ISSUE TITLE
About noise difference in video & image generation

### ISSUE THREAD

**Issue by XuWuLingYu** [cure-lab/MagicDrive#138]
In image generation, different views use the same noise as validation initial.
`torch.stack([latents]*6,dim=0)`
But in the video generation, it seems not work to do the same way. The nosie between views must be different. Is there any research can tell why?

**Comment by flymin** [comment]
We did not strictly verify these options.

We did try training image generation by adding the same noise to different views and it does not work. 

I think the expected operation may be use different noise to each single view, no matter multi-view or video generation. Thus, it may not be necessary to use the same initial noise for image generation.
